.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: java
+
+jdk:
+ - openjdk8
+ 
+ cache:
+  directories:
+  - $HOME/.m2
+
+script:
+  - mvn clean install
+  - mvn clean test
```

Comment:
these 2 are running tests and so no need of the second

Reply:
can you lauch a docker ref app instance and point the tests properties to it in here

Reply:
Some more clarification about this, Are you meaning of testing this commit against running docker  refapp instance

Reply:
This may not work when trying to test it out , not sure if i answer your question thanks

Reply:
you need the first of those 2 commands. and yes, we need to lauch ref app within travis, wait fo rit and test against it

Reply:
Have only included only one command thanks

Reply:
Seems am mistakenly understanding your point, isnt this only testing the build only